src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/DraftAdjudicationDto.kt

```diff
@@ -38,6 +38,8 @@ data class IncidentDetailsDto(
 data class IncidentRoleDto(
   @ApiModelProperty(value = "The incident role code", notes = "If not set then it is assumed they committed the offence on their own", example = "25a")
   val roleCode: String?,
+  @ApiModelProperty(value = "The offence rules related to the given incident role", notes = "Will not be set of there is no role code")
```

Comment:
By this do you mean the role code has not yet been set - only reason I ask is cause the role code can be null when the prisoner committed the offence?